src/Acme.UI/Properties/PublishProfiles/LocalDev.pubxml.user

```diff
@@ -13,19 +13,19 @@ by editing this MSBuild file. In order to learn more about this please visit htt
       <publishTime>08/08/2019 13:12:23</publishTime>
```

Comment:
In your next commit can you please get rid of pubxml.user files?
They seem to constantly update this isnt providing any value in source control